addons/test_mail/tests/test_mail_followers.py

```diff
@@ -766,3 +772,312 @@ def test_recipients_fetch(self):
             pids=test_partners.ids
         )
         self.assertRecipientsData(recipients_data, False, test_partners)
+
+
+class UnfollowUnreadableRecordTest(MailCommon):
```

Comment:
Would add tagged('mail_followers'), so that we can launch all followers tests easily :)

Reply:
Yes, good idea.